packages/example/src/components/Connect/Browser.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Modal } from "semantic-ui-react";
+import { Icon } from "snapkit";
+import CloseIcon from "../Icons/CloseIcon";
+
+interface BrowserProps {
+  open: boolean;
+  close: () => void;
+}
+
+const Browser = ({open, close}: BrowserProps) => {
+
+  return (
+    <Modal open={open} style={{width: 552, height: 333, borderRadius: 20}}>
+      <div className="Browser-Container"><CloseIcon onClick={close} /></div>
+      <div className='sk-t-center'>
+        <div className='sk-flex sk-justify-center browsers'>
+          <a className='sk-bg-hover-ntd04 sk-a-transition' href='https://www.google.com/chrome/'>
+            <Icon.Chrome></Icon.Chrome>
+            <div className='sk-t-h5 sk-c-n60'>Chrome</div>
+          </a>
+          <a className='sk-bg-hover-ntd04 sk-a-transition' href='https://www.mozilla.org/'>
+            <Icon.Firefox></Icon.Firefox>
+            <div className='sk-t-h5 sk-c-n60'>Firefox</div>
+          </a>
+          <a className='sk-bg-hover-ntd04 sk-a-transition' href='https://brave.com/'>
+            <Icon.Brave></Icon.Brave>
+            <div className='sk-t-h5 sk-c-n60'>Brave</div>
+          </a>
+          <a className='sk-bg-hover-ntd04 sk-a-transition' href='https://www.microsoft.com/en-us/edge'>
+            <Icon.Edge></Icon.Edge>
+            <div className='sk-t-h5 sk-c-n60'>Edge</div>
+          </a>
+        </div>
+        <div className='sk-t-h2 sk-c-n80 msg'>This Browser is not Supported</div>
+        <div className='sk-t-caption sk-c-n60 desc'>Please use the browsers above we currently supported.</div>
+      </div>
```

Comment:
May be we need a component from snapkit here, instead of handling all details ourselves.